cognite/client/data_classes/capabilities.py

```diff
@@ -176,31 +187,39 @@ def load(cls, resource: dict | str, allow_unknown: bool = False) -> Self:
             (name,) = known_acls
             capability_cls = _CAPABILITY_CLASS_BY_NAME[name]
             # TODO: We ignore extra keys that might be present like 'projectUrlNames' - are these needed?
+            try:
+                scopes = Capability.Scope.load(resource[name]["scope"], allow_unknown)
+            except Exception as err:
+                raise ValueError(f"Could not instantiate {capability_cls.__name__} due to: {err}") from None
```

Comment:
why not like this? In my experience, getting the traceback for the wrapped exception is more useful than just getting the `exception.message` field.
```suggestion
                raise ValueError(f"Could not instantiate {capability_cls.__name__}") from err
```

Reply:
Yeah, I agree. But I think I should have just not used the from-syntax at all (the first error is not the direct cause of the one I'm raising, right). Will fix